swap/config.go

```diff
@@ -33,7 +33,6 @@ const (
 	// This is the amount of time in seconds which an issuer has to wait to decrease the harddeposit of a beneficiary.
 	// The smart-contract allows for setting this variable differently per beneficiary
 	defaultHarddepositTimeoutDuration = 24 * time.Hour
-
-	// While Swap is unstable, it's only allowed to be run under a specific network ID
+	// While Swap is unstable, it's only allowed to be run under a specific network ID (use the --bzznetworkid flag to set it)
```

Comment:
Not from this PR, but as you are changing, let's not say "unstable". This flag is not about being unstable, but until we have a pricing which makes sense. 

Let's say "Until we deploy swap officially, it's only allowed to be enabled under a ....."

Reply:
Will change